rllib/core/learner/learner_group.py

```diff
@@ -150,11 +164,18 @@ def __init__(
                 "GPU": num_gpus_per_learner,
             }
 
+            # Create a placement, if necessary.
+            self._placement_group = self._create_placement_group(
```

Comment:
nit: Let's create a property and then use that property in `Algorithm` (instead of the private attr):

```
    @property
    def placement_group(self) -> Optional[PlacementGroup]:
        return self._placement_group
```